.gitignore

```diff
@@ -44,3 +44,4 @@ test/fixtures/*/terraform.tfvars
 terraform.tfstate.d/
 *.auto.tfvars
 credentials.json
+/.idea/
```

Comment:
Not sure that we need it

Reply:
We mostly use Idea as the editor, so we need it